tests/integration/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required( VERSION 3.5 )
 
-set(EOSIO_VERSION_MIN "3.1")
+set(EOSIO_VERSION_MIN "1.0")
```

Comment:
Should it still be called `EOSIO_VERSION`, now that it goes back to 1.0. But no need to change for now.